go.mod

```diff
@@ -3,17 +3,21 @@ module github.com/openshift/sippy
 go 1.12
 
 require (
+	cloud.google.com/go/bigquery v1.8.0
 	github.com/GeertJohan/go.rice v1.0.2
 	github.com/akavel/rsrc v0.10.2 // indirect
 	github.com/daaku/go.zipexe v1.0.1 // indirect
 	github.com/elastic/go-elasticsearch/v7 v7.5.1-0.20210823155509-845c8efe54a7
+	github.com/googleapis/gax-go v1.0.3 // indirect
 	github.com/jessevdk/go-flags v1.5.0 // indirect
 	github.com/montanaflynn/stats v0.6.6
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v0.0.6
 	github.com/tidwall/gjson v1.9.4
 	github.com/valyala/fasttemplate v1.2.1 // indirect
 	go.etcd.io/bbolt v1.3.6 // indirect
-	golang.org/x/sys v0.0.0-20211004093028-2c5d950f24ef // indirect
+	google.golang.org/api v0.60.0
+	gorm.io/driver/postgres v1.2.1
+	gorm.io/gorm v1.22.2
```

Comment:
I'd been tempoted to rely mostly on jsonb columns, but doing a little research, this is probably better. There's a price to pay for use of json in terms of what you can do with the data and for things we control internal to sippy itself, it makes sense to do proper relational db. hopefully gorm makes that fairly easy.

Reply:
gorm can do jsonb, https://v1.gorm.io/docs/dialects.html -- so we could always use that to start for existing reports, and move to normalized data over time

Reply:
That's really cool, and might be a good idea, we can treat the db as totally throwaway for some time while we sort that all out.